WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.kt

```diff
@@ -264,6 +264,9 @@ class PostsListActivity : LocaleAwareActivity(),
             }
         })
 
+        // Notification opens in Drafts tab
+        if (actionsShownByDefault) tabLayout.getTabAt(PostListType.DRAFTS.ordinal)?.select()
```

Comment:
Instead of making this behavior implicit through the reuse of the `actionsShownByDefault` parameter, I'd rather make it explicit by either renaming the parameter or adding a new one. That way, we would have a better idea of how the parameter changes the screen behavior when building a new intent for it. Does that make sense? Let me know what you think!

Reply:
Makes sense, update the code to introduce a new param for tab index based on PostListType.  Thanks